### PR TITLE
Logo and readme

### PR DESCRIPTION
**Pull request by pioardi**
## PR Checklist

- [x] Please add a description of your changes.
- [x] We need your changes to come with unit tests in order to keep this project in quality and easy to maintain.
- [ ] Please add a link to the open issue or task that this pull request will resolve.
- [ ] Add a note to the changelog to indicate the change.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

If yes please indicate it and migration info into the CHANGELOG.md file.

---

This PR added the logo to our readme and several badges about sonar ( quality gate is unfortunately in fail state but I am sure will be soon green). I added a list of our main pillars and reason to choose this project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,17 +4,43 @@
 [![Dependabot](https://badgen.net/dependabot/dependabot/dependabot-core/?icon=dependabot)](https://badgen.net/dependabot/dependabot/dependabot-core/?icon=dependabot)
 [![npm w](https://img.shields.io/npm/dw/poolifier)](https://www.npmjs.com/package/poolifier)
 [![Actions Status](https://github.com/pioardi/node-pool/workflows/NodeCI/badge.svg)](https://github.com/pioardi/node-pool/actions)
-[![Coverage Status](https://coveralls.io/repos/github/pioardi/poolifier/badge.svg?branch=master)](https://coveralls.io/github/pioardi/poolifier?branch=master)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=pioardi_poolifier&metric=alert_status)](https://sonarcloud.io/dashboard?id=pioardi_poolifier)
+ [![SonarCloud Coverage](https://sonarcloud.io/api/project_badges/measure?project=pioardi_poolifier&metric=coverage)](https://sonarcloud.io/component_measures/metric/coverage/list)
```
Comment (jerome-benoit) [inline README.md:8]: The second link does not work.

Comment (pioardi) [reply README.md]: Yeah just fixed now, please give it a try jerome

**Comment by pioardi** [comment]
@jerome-benoit  @Shinigami92  behind the PR I would like to have feedbacks on our list of "why use poolifier", please suggest any further feature/highlight that we can add in that list or even a change with another already there.

**Comment by Shinigami92** [comment]
@pioardi
- You should center the logo
- The logo is a bit to big in size
- You can also center the badges too
- Behind security the two badges need a space in between 
image
- `No runtime dependencies white_check_mark (until now we don't have any exception to that)` could be better `No runtime dependencies :heavy_check_mark: (until now we don't have any expectations to add some)`
- Active community -> "There are three active maintainers" or similar
- Maybe the whole list needs a better representation / design? Maybe a table or so?!

---

@ST-DDT Do you also want to have a look as an outside person?

**Comment by Shinigami92** [comment]
File can be viewed rawly here: https://github.com/pioardi/poolifier/blob/logo-and-readme/README.md

**Comment by pioardi** [comment]
> @pioardi
> 
> * You should center the logo
> * The logo is a bit to big in size
> * You can also center the badges too
> * Behind security the two badges need a space in between
>   image
> * `No runtime dependencies white_check_mark (until now we don't have any exception to that)` could be better `No runtime dependencies :heavy_check_mark: (until now we don't have any expectations to add some)`
> * Active community -> "There are three active maintainers" or similar
> * Maybe the whole list needs a better representation / design? Maybe a table or so?!
> 
> @ST-DDT Do you also want to have a look as an outside person?

I decreased logo size and centered the logo
I change also the point on runtime dependencies and added a space between the two security badges
On active community we really don't need to say how much members are into it, we are an active community anyway

I would need your help to center badges and find a better representation for the list

**Comment by Shinigami92** [comment]
You need html as a fallback for that
You can look into my plugin https://raw.githubusercontent.com/prettier/plugin-pug/main/README.md

The logo now has a good size :+1:

**Comment by ST-DDT** [comment]
The centered logo looks good to me. And I would probably also center the badges and the title (not sure whether a centered title looks good/intuitive, but badge and title alignment should be the same). 

However I would reorder the badges to match their group/importance/relevance for external users. Suggestion:
- Downloads
- NodeCI
- Coverage
- QualityGamte
- Gitter
- PRswelcome
- Deps+Deps Bot

File: README.md [hunk 2]
```diff
@@ -4,17 +4,43 @@
...
 ## Why Poolifier?
 
 Poolifier is used to perform heavy CPU bound tasks on nodejs servers, it implements thread pools (yes, more thread pool implementations, so you can choose which one fit better for you) using [worker-threads](https://nodejs.org/api/worker_threads.html#worker_threads_worker_threads).  
 With poolifier you can improve your **performance** and resolve problems related to the event loop.  
 Moreover you can execute your CPU tasks using an API designed to improve the **developer experience**.
 
+- Performance :racehorse:
+- Security :bank: :cop: [![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=pioardi_poolifier&metric=security_rating)](https://sonarcloud.io/dashboard?id=pioardi_poolifier) [![Vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=pioardi_poolifier&metric=vulnerabilities)](https://sonarcloud.io/dashboard?id=pioardi_poolifier)
+
+- No runtime dependencies :heavy_check_mark: (until now we don't have any expectations to add some)
```
Comment (ST-DDT) [inline README.md:26]: I would remove the remark at the end

File: README.md [hunk 3]
```diff
@@ -4,17 +4,43 @@
...
+- Easy to use :couple:
```
Comment (ST-DDT) [inline README.md:27]: This one is very important, move it to the top

File: README.md [hunk 4]
```diff
@@ -4,17 +4,43 @@
...
+[![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=pioardi_poolifier&metric=sqale_index)](https://sonarcloud.io/dashboard?id=pioardi_poolifier)
+- Easy switch from a pool to another, easy to tune
+- Dynamic pool size
+- Proper async integration with node async hooks
+- Support for worker threads and cluster node modules
+- Active community
+- Support sync and async tasks
+- General guidance on pools to use
+- Widely tested
+- Error handling out of the box
```
Comment (ST-DDT) [inline README.md:42]: These are important to. (The users of a library don't care (much) about the code quality, as long as it "works as expected")

File: README.md (hunk 2 above)
Comment (pioardi) [reply README.md:26]: yes maybe better to keep it more concise

File: README.md (hunk 4 above)
Comment (pioardi) [reply README.md:42]: So this is a good consideration, I will keep this but at the end of the list is maybe better

**Comment by Shinigami92** [comment]
@pioardi looking MUCH better now

File: README.md (hunk 3 above)
Comment (pioardi) [reply README.md:40]: Thank you thank you thank you, moved at the third position after security and performance

**Comment by pioardi** [comment]
> @pioardi looking MUCH better now

Ohhh yeah bro, this is the result when we use feedback loops :) It is one of my main mantra also during paid job

**Comment by pioardi** [comment]
@Shinigami92  if you approv, I guess we are ready to merge this

**Review by Shinigami92 — APPROVED** [review]

**Comment by pioardi** [comment]
Thanks also to @ST-DDT, I will include you soon into our readme as PR reviewer(together with some other occasional contributors)

**Pull request merged**